src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2012,8 +2021,38 @@ private bool IsTypeDeclarationStart()
             }
         }
 
+        public MemberDeclarationSyntax ParseMemberDeclaration()
```

Comment:
>ParseMemberDeclaration [](start = 39, length = 22)

Should similar API be added to VB?

Reply:
That would be a completely separate feature request for which we have not had any customer request.

Reply:
> That would be a completely separate feature request for which we have not had any customer request.

I do not see anything C# specific in customer scenario. Please get explicit confirmation from Public API Design Team that we don't want to add similar API to VB at the same time.

---
In reply to: [210053701](https://github.com/dotnet/roslyn/pull/29263#discussion_r210053701) [](ancestors = 210053701)

Reply:
It is confirmed that the API as specified here (i.e. implemented in C# only and not VB) has been approved.

---
In reply to: [210344157](https://github.com/dotnet/roslyn/pull/29263#discussion_r210344157) [](ancestors = 210344157,210053701)